control-plane/outputs.tf

```diff
@@ -137,6 +137,10 @@ output "secrets_encryption" {
   value = var.enable_secrets_encryption
 }
 
+output "cattle_prometheus_metrics" {
+  value = var.cattle_prometheus_metrics
+}
+
 output "use_new_monitoring_crd_url" {
-  value = module.k3s[0].use_new_monitoring_crd_url
-}
\ No newline at end of file
+  value = var.k8s_distribution == "k3s" && var.install_monitoring ? module.k3s[0].use_new_monitoring_crd_url : null
```

Comment:
> Could we add a comment behind the reasoning of the logic of this code? Just in case someone else picks up enhancing or extending these scripts.

This was more of a debugging-oriented output to confirm that the right github url was used for the aws-k3s module. Essentially, the [2.5 rancher/charts branches](https://github.com/rancher/charts/tree/dev-v2.5/assets) dont have a url for `rancher-monitoring-crd` assets directory, whereas the [2.6 rancher/charts branches](https://github.com/rancher/charts/tree/dev-v2.6/assets) do.

This output can probably just be removed from the top-level outputs.tf